website/docs/getting-started/building-an-image.md

```diff
@@ -16,14 +16,24 @@ With Podman Desktop, you can build an image from a Containerfile on your contain
 
 #### Procedure
 
-1. Open **<icon icon="fa-solid fa-cloud" size="lg" /> Images**.
+1. Go to **<icon icon="fa-solid fa-cloud" size="lg" /> Images**.
 1. Click **<icon icon="fa-solid fa-cube" size="lg" /> Build an image**.
 1. On the **Build Image from Containerfile** screen
-   1. **Containerfile path**: select the Containerfile to build.
-   1. **Image Name**: edit the image name: `my_image`.
+   1. **Containerfile path**: select the `Containerfile` or `Dockerfile` to build.
+   1. **Image Name**: enter your image name, such as `my_image`. If you want to push the image to a registry, use the fully qualified image name that your registry requires, such as `my_registry.tld/my_repository/my_image`.
```

Comment:
what about using real example like quay.io or ghcr.io or docker.io as example instead of my_registry.tld ?

I think underscore is not so used so hyphens might be preferable like `my-image` instead of `my_image`, repository instead of `my_repository`

Reply:
For the underscores: my mistake, automated-habit, it's the general rule we have for placeholders.

Reply:
I thought about a real example, but it is a repository the user should be able to write to: I am worried it will make things more complicated.